modules/squants/build.sbt

```diff
@@ -3,7 +3,7 @@ name := "pureconfig-squants"
 crossScalaVersions ~= { _.filterNot(_.startsWith("2.13")) }
 
 libraryDependencies ++= Seq(
-  "org.typelevel" %% "squants" % "1.3.0")  // blocked by https://github.com/typelevel/squants/issues/321
+  "org.typelevel" %% "squants" % "1.6.0")  // blocked by https://github.com/typelevel/squants/issues/321
```

Comment:
Since we're using our own string representation for `Money`, we're no longer blocked on 1.3.0.